front-end/src/themes/theme.ts

```diff
@@ -0,0 +1,13 @@
+export const theme = {
+	colors: {
+		black01: '#282828',
+		primary: '#FF5A47',
```

Comment:
Having a `black` and then `primary` is weird., I'd go for something a little more explicit, like `primary-red` because having a `background-color: primary` then feels a little too general to me.

Reply:
Sure, agree